src/Launcher/LauncherInterface.php

```diff
@@ -9,10 +9,163 @@
 use Drupal\Component\Plugin\ConfigurablePluginInterface;
 use Drupal\Component\Plugin\PluginInspectionInterface;
 use Drupal\Core\Plugin\PluginFormInterface;
+use Drupal\ultimate_cron\Entity\CronJob;
 
 /**
  * Defines a launcher method.
  */
 interface LauncherInterface extends PluginInspectionInterface, ConfigurablePluginInterface, PluginFormInterface {
 
+  /**
+   * Default settings.
+   */
+  public function defaultConfiguration();
```

Comment:
This should have a return array description. Are we really calling this from the outside?